cli/mender-artifact/debugfs.go

```diff
@@ -73,6 +73,22 @@ func debugfsCopyFile(file, image string) (ret string, err error) {
 	return tmpDir, nil
 }
 
+func debugfsMakeDir(imageFile, image string) (err error) {
+	// Remove the `/` suffix if present, as debugfs does not play nice with it.
+	imageFile = strings.TrimRight(imageFile, "/")
+	// Recursively create parent directories if they do not exist
+	dir, _ := filepath.Split(imageFile)
+	_, err = os.Stat(dir)
+	if dir != "" && os.IsNotExist(err) {
+		debugfsMakeDir(dir, image)
```

Comment:
I don't quite follow this code. Isn't `imageFile` the path inside of the image? So why is `dir` being tested using `os.Stat` on the host filesystem?

Reply:
Good catch! The UNIX `install -d` silently does nothing if the directory already exists, so I'll mirror that behavior.

Reply:
I removed this `os.Stat` logic and replaced it with a `debugfs stat` check to simply return if the directory already exists.